src/test/fuzz/versionbits.cpp

```diff
@@ -323,7 +322,7 @@ FUZZ_TARGET_INIT(versionbits, initialize)
         assert(exp_state == ThresholdState::ACTIVE || exp_state == ThresholdState::LOCKED_IN);
         break;
     case ThresholdState::FAILED:
-        assert(current_block->GetMedianTimePast() >= checker.m_end);
+        assert(never_active_test || current_block->GetMedianTimePast() >= checker.m_end);
```

Comment:
The `if (never_active_test)` block at the end asserts both `state` and `exp_state` are always FAILED so that's already covered (likewise for `always_active_test`).